<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPanorama</class>
 <widget class="QWidget" name="VideoPanorama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1898</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(12, 22, 36);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameBorder">
     <property name="styleSheet">
      <string notr="true">QFrame#frameBorder {
    /* 2像素的青蓝色实线边框 */
    border: 2px solid rgb(0, 160, 230); 
    /* 纯黑背景，模拟视频未播放时的状态 */
    background-color: black; 
    /* 稍微圆角一点点 */
    border-radius: 4px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QStackedWidget" name="stackVideoMode">
        <widget class="QWidget" name="page">
         <widget class="QWidget" name="pagePanorama" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1872</width>
            <height>562</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="pageMultiCam" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
